UI/Web/src/app/book-reader/_components/book-reader/book-reader.component.ts

```diff
@@ -475,8 +477,14 @@ export class BookReaderComponent implements OnInit, AfterViewInit, OnDestroy {
         this.handleScrollEvent();
     });
 
-    // TODO: In order to allow people to highlight content under pagination overlays, apply pointer-events: none while
-    // a highlight operation is in effect
+    fromEvent<MouseEvent>(this.bookContainerElemRef.nativeElement, 'mousemove')
+      .pipe(
+        takeUntilDestroyed(this.destroyRef),
+        map(this.mapMousePositionToPointerCursor.bind(this)),
+        distinctUntilChanged())
+      .subscribe((pointer) => {
+        this.changeCursor(pointer);
```

Comment:
This can be rewritten:
    fromEvent<MouseEvent>(this.bookContainerElemRef.nativeElement, 'mousemove')
      .pipe(
        takeUntilDestroyed(this.destroyRef),
        map(() => this.mapMousePositionToPointerCursor)),
        distinctUntilChanged(),
        tap((pointer) => this.changeCursor(pointer))
      .subscribe()